src/payloads/range-timeseries-data.ts

```diff
@@ -0,0 +1,23 @@
+export interface RangeData {
+  data: RangeDefinition[];
+  timeSeries: number[];
+  chartingMetadata: ChartingMetadata;
+}
+
+export interface ChartingMetadata {
+  title: string;
+  xAxisLabel: string;
+  yAxisLabel: string;
+}
+
+export interface RangeDefinition {
+  upper: Bound;
+  lower: Bound;
+  average: Bound;
```

Comment:
the backend should just return 5 time series. And the frontend turn them into 2 arearanges and one time series.

Reply:
Then next time if we want to change the front-end visualization, we don't need to change the backend.